docs/_writing-tests/general-guidelines.md

```diff
@@ -112,6 +112,32 @@ either installed or to have specific metrics. As such, in most cases
 when a known font is needed Ahem should be used. In other cases,
 `@font-face` should be used.
 
+#### Be Conservative
+
+Sometimes, when authoring tests for one web platform feature, it may be
+convenient to use some other feature (e.g. a web browser API or ECMAScript
+language feature) that is not directly related to the test itself. Reliance on
+such extraneous features tends to make tests more difficult to consume, so care
+should be taken when considering their use.  Tests may only rely on on language
+features that are available in the latest version of all of the following
+browsers:
+
+- [Mozilla Firefox][mozilla-firefox]
+- [Google Chrome][google-chrome]
+- [Apple Safari][apple-safari]
+- [Microsoft Edge][microsoft-edge]
+
+For JavaScript code that is re-used across many tests (e.g. `testharness.js`
```

Comment:
Let's just keep the `testharness.js` exception (and I think it is the only exception) documented in the header of it, and not have it here. I think it's sufficiently obscure that it needn't be here.